client/layout/transient-notices/style.scss

```diff
@@ -34,4 +38,9 @@
 		top: 24px;
 		left: 26px;
 	}
+
+	.components-snackbar__explicit-dismiss {
```

Comment:
It's confusing having `.components-snackbar-explicit-dismiss` and `components-snackbar__explicit-dismiss`. How about naming this one something like `components-snackbar__dismiss-button`?

Reply:
Nice, done in 09de3fd8e9c807290f5305780f71624dca6b99e2